front/app/containers/Admin/dashboard/summary/charts/BarChartActiveUsersByTime.tsx

```diff
@@ -55,6 +56,8 @@ const StyledResponsiveContainer = styled(ResponsiveContainer)`
   }
 `;
 
+const offset = new Date().getTimezoneOffset();
```

Comment:
I think you can remove that part, default moment constructor offesets for current timezone already https://momentjs.com/docs/#/parsing/, https://maggiepint.com/2016/05/14/moment-js-shows-the-wrong-date/

Reply:
Indeed, nice observation! I replaced the explicit offset I was passing with the native moment utc function and it seems to do the trick!